spec/recipe.fmf

```diff
@@ -0,0 +1,109 @@
+story: |
+    As a user, I want tmt to generate a recipe file that can be reused to reproduce
+    a run with the exact same configuration, without requiring additional input.
+
+description: |
+    A recipe is a YAML file that captures a complete, static snapshot of a tmt
+    run after all dynamic evaluation has been resolved. It includes preprocessed
+    information about plans, tests, results, and run configuration. All environment
+    variables are evaluated, and git references are replaced with fixed commit hashes.
+
+    Generated recipe can be edited, manually or by using a script, which allows users
+    to select only a subset of discovered tests to be reexecuted.
+
+    .. code-block:: yaml
```

Comment:
I would strongly recommend following the type/description documented format from Results and HW requirement specification:

```yaml
.. code-block:: yaml

    # Mapping, stores git repository configuration
    git:
      # String, URL of the repository.
      url: http://some.git.host.com/project/tests.git
      # String, git reference - branch, commit, tag - to check out in the repository.
      ref: main
      # String, path to the fmf root in the repository.
      path: /some/path
```

And so on.